src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -154,14 +161,15 @@ const TrackPanelListItem = (props: TrackPanelListItemProps) => {
     browserMessagingService.send('bpane', payload);
   };
 
-  const listItemClassNames = classNames(styles.listItem, {
-    [styles.main]: track.track_id === 'gene',
-    [styles.currentDrawerView]: track.track_id === drawerView
+  const trackClassNames = classNames(styles.track, {
+    [styles.main]: track.track_id === 'track:gene-feat',
```

Comment:
We should be using track name constants instead of magic strings. This is probably related to the comment I've made above.

Reply:
That's done